src/services/iasql.ts

```diff
@@ -188,23 +189,30 @@ export async function runSql(dbAlias: string, uid: string, sql: string) {
       ssl: dbMan.baseConnConfig.extra.ssl,
     });
     await connTemp.connect();
-    const out = await connTemp.query(sql);
+    const stmts = parse(sql);
+    const out = [];
+    for (const stmt of stmts) {
+      out.push(await connTemp.query(deparse(stmt)));
+    }
+    // const out = await connTemp.query(sql);
     // Let's make this a bit easier to parse. Error -> error path, single table -> array of objects,
     // multiple tables -> array of array of objects
-    if (typeof out === 'string') {
-      return out;
-    } else if (
-      !!out.rows &&
-      out.rows.length === 0 &&
-      out.command !== 'SELECT' &&
-      typeof out.rowCount === 'number'
-    ) {
-      return [{ affected_records: out.rowCount, }];
-    } else if (!!out.rows) {
-      return out.rows;
-    } else if (out instanceof Array) {
-      return out.map(t => t.rows);
-    }
+    return out.map(t => {
+      if (
+        !!t.rows &&
+        t.rows.length === 0 &&
+        t.command !== 'SELECT' &&
+        typeof t.rowCount === 'number'
+      ) {
+        return ({ affected_records: t.rowCount, });
+      } else if (typeof t === 'string') {
+        return ({ result: t, });
+      } else if (!!t.rows) {
+        return t.rows;
+      } else {
+        return ({ error: 'unexpected result: ${t}', }); // TODO: Error this out
+      }
+    });
```

Comment:
The screenshot shows that it knows how to deal with it, though. We already had the dashboard updated.

I am tempted to change the output to an array of objects with `"statement": <statement>, "result": <rows>,` as the contents, though, which would break the dashboard, but allow us to display *all* statement outputs more clearly.

I figured I'd do that in a different PR.

Reply:
![Screenshot from 2022-07-28 09-46-23](https://user-images.githubusercontent.com/765551/181572889-ac2ac2d1-803b-4848-885c-3bf8fdeae342.png)

The picture in question. Forgot to attach to this PR

Reply:
Could you try the case where a failure occurs? that is when the `typeof string` condition occurs, I'm not sure if the dashboard will handle that case

Reply:
![Screenshot from 2022-07-28 10-26-48](https://user-images.githubusercontent.com/765551/181576884-f461be9c-2bad-4043-8ade-6277e4a20bb1.png)

It works :)